data/datasets/corruptedData/sectionAnomaly/README.md

```diff
@@ -0,0 +1,8 @@
+# Corrupted data - section anomaly
+
```

Comment:
I was wondering whether we should include this in readme.md in `.../sectionAnomaly/` folder. It seems more logical to me.